app/controllers/users_controller.rb

```diff
@@ -1,11 +1,11 @@
 class UsersController < ApplicationController
-  def index; end
-
-  def show
-    @id = params[:id]
+  def index
+    @users = User.all
   end
 
-  def posts
-    @id = params[:user_id]
+  def show
+    user_id = params[:id].to_i
+    @user = User.find(user_id)
+    @posts = @user.recent_posts
```

Comment:
- [ ] As requested by the previous reviewer that your app is giving that issue with the recent_post methods in the user_show view :thinking: Kindly fix that because I am facing the same issue and your app doesn't go further :fearful: